rotkehlchen/exchanges/manager.py

```diff
@@ -306,11 +306,29 @@ def get_user_binance_pairs(self, name: str, location: Location) -> list[str]:
             return self.database.get_binance_pairs(name, location)
         return []
 
-    def query_history_events(self) -> None:
-        """Queries all history events for exchanges that need it
+    def query_history_events(self, location: Location, name: str | None) -> None:
```

Comment:
note that this changes the basic naming rules of this class. As every type of event was separate to query_online_XXX and query_XXX. This new thing you are making here breaks this. How would you document this for the interim? Moving everything to history events will take time.